### PR TITLE
Unit addresses

### PR DESCRIPTION
**Pull request by steve-thorne-ONS**
Unit address changes and validation.

### CODE REVIEW

File: src/main/java/uk/gov/ons/entities/HybridAddress.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+@AllArgsConstructor
+public @Data class HybridAddress {
+
+	@Id
+	private Long uprn;
+	private String classificationCode;
+	private String censusAddressType;
+	private String censusEstabType;
+//	private Long censusEstabUprn;
```
Comment (sdiaz1000) [inline src/main/java/uk/gov/ons/entities/HybridAddress.java:17]: phew found something. Remove m=comments :-)

File: src/main/java/uk/gov/ons/entities/HybridAddress.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	private String postcode;
+	private CountryCode countryCode;
+//	private String postcodeStreetTown;
```
Comment (sdiaz1000) [inline src/main/java/uk/gov/ons/entities/HybridAddress.java:20]: ditto

File: src/main/java/uk/gov/ons/entities/InputAddress.java [hunk 3]
```diff
@@ -18,46 +21,68 @@
 public @Data class InputAddress {
 
 	@CsvBindByName(column = "address_line1")
+	@NotBlank(message = "ADDRESS_LINE1 is mandatory")
```
Comment (sdiaz1000) [inline src/main/java/uk/gov/ons/entities/InputAddress.java:24]: I find it difficult to read with out blank line inbetween variables, but up to u. consistency

**Review by sdiaz1000 — APPROVED** [review]

File: src/main/java/uk/gov/ons/entities/HybridAddress.java (hunk 1 above)
Comment (steve-thorne-ONS) [reply src/main/java/uk/gov/ons/entities/HybridAddress.java:17]: Yes I don't like leaving in commented out code. I think these will be required in the next code change so I just kept them.

**Pull request merged**